_activities/forcesfriction.md

```diff
@@ -5,12 +5,27 @@ lang: en
 # Content
 #
 title: "Forces and Friction"
-title_fr:
+title_fr:"Forces et Friction"
 age: 8-12
-objective: "Learn how forces affects the motion of objects."
-objective_fr:
-summary:
-summary_fr:
+objective: "Learn how forces affect the motion of objects."
+objective_fr:"Apprendre comment que les forces influencent le mouvement d'un objet."
+summary:"In this activity, students will perform experiments using a pulley-weight setup in order to understand the effects of forces on the motion of an object. They will learn about how the force applied on the object and the mass of the object affects it's motion by adding washers to both ends of a string that is attached to a pulley.  They will also experiment with the force of friction acting on the object, and discover how a sandpaper surface affects the object’s motion."
+summary_fr:"Dans cette activité, les élèves vont performer des expériences en utilisant un système de poulie et poids afin de comprendre comment que les forces influencent le mouvement d’un objet. Ils vont apprendre comment que la masse affecte le mouvement de l’objet en ajoutant des rondelles au deux extrémités de la corde qui est attachée à la poulie. Ils vont aussi expérimenter la force de friction qui agit sur un objet, puis ils vont découvrir comment que du papier de verre affecte le mouvement de l’objet.    "
+image:
+  thumb: thumb_forcesfriction.jpg
+  title: forcesfriction.jpg
+  alttext: "Forces and Friction Setup"
+  alttext_fr: "Installation Forces et Friction"
+downloads:
+  -
+    name: "Facilitator guide - English"
+    filename: "Forces and Friction_facilitator_guide.pdf"
+  -
+    name: "Student worksheet - English"
+    filename: "Forces and Friction_student_worksheet_EN.pdf"
+  -
+    name: "Fiche d'activité de l'élève - français"
+    filename: "Forces and Friction_student_worksheet_FR.pdf"
```

Comment:
Filename in downloads folder: "Forces and friction - Student Worksheet_FR.pdf" (Sorry these are not consistent! It would have been easier if i was more careful in the naming.)